terraform/modules/kubernetes/azure_redis.tf

```diff
@@ -59,6 +59,18 @@ resource "azurerm_private_endpoint" "redis-cache-private-endpoint" {
   }
 }
 
+resource "azurerm_storage_account" "redis_cache_backup" {
+  count = var.deploy_azure_backing_services ? 1 : 0
+
+  name                     = "${var.azure_resource_prefix}${var.service_short}rdbkp${var.config_short}sa"
```

Comment:
I would define the name as a local in variables.tf. Maybe use either `rds` or `redis` instead of `rd` as per: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations